app/views/users/login.erb

```diff
@@ -0,0 +1,9 @@
+<form method="post" action="/users/login">
+    <label for="user_name_input">Username: </label>
+    <input id="user_name_input" name="user[name]" type="text" value="">
+
+    <label for="user_password_hash_input">Password: </label>
+    <input id="user_password_hash_input" name="user[password_hash]" type="text" value="">
```

Comment:
When we implement BCrypt, change 
`name="user[password_hash]"`
 to 
`name=user[password]"`